wagtail/contrib/redirects/templates/wagtailredirects/confirm_import.html

```diff
@@ -0,0 +1,55 @@
+{% extends "wagtailadmin/base.html" %}
+{% load i18n %}
+
+{% block titletag %}{% trans "Confirm import" %}{% endblock %}
+{% block content %}
+    {% trans "Confirm import" as confirm_str %}
+    {% include "wagtailadmin/shared/header.html" with title=confirm_str icon="redirect" %}
```

Comment:
Similar to the other headers, having a consistent main header and adding a subtitle may provide a bit more of a unified feel to this workflow.

```
    {% trans "Import redirects" as header_title %}
    {% trans "Confirm import" as header_subtitle %}
    {% include "wagtailadmin/shared/header.html" with title=header_title subtitle=header_subtitle icon="redirect" %}
```